dbsnp-importer/src/test/resources/data.sql

```diff
@@ -14,6 +15,9 @@ INSERT INTO snp VALUES(13808689, 0.5, 0, '2004-07-02 16:03:00.000', '2015-03-27
 INSERT INTO snp VALUES(739617577, NULL, NULL, '2015-03-25 15:37:00.000', '2015-03-25 15:36:57.023', NULL, 9031, NULL, 1540359250, 238, 1, NULL);
 INSERT INTO snp VALUES(13511401, 0.625, 0.125, '2004-07-02 16:03:00.000', '2017-05-05 15:41:16.507', NULL, 9031, 3, 191122885, 146, 9, NULL);
 INSERT INTO snp VALUES(733889725, NULL, NULL, '2015-03-25 15:37:00.000', '2017-05-05 14:55:28.727', NULL, 9031, 1, 1394569659, 25, 3, NULL);
+INSERT INTO snp values ('3136863','0.5','0','2002-04-10 21:07:00','2017-05-05 16:34:08.087',NULL,9031,3,'4387083',25,9,NULL)
```

Comment:
There are several numbers in all the new added insert statements that are surronded by quotes, like snp and subsnp ids.  I'm not sure if those values are inserted correctly, but I think the quotes should be removed to have an uniform data set